cluster-autoscaler/processors/processors.go

```diff
@@ -31,14 +31,18 @@ type AutoscalingProcessors struct {
 	NodeGroupListProcessor nodegroups.NodeGroupListProcessor
 	// ScaleUpStatusProcessor is used to process the state of the cluster after a scale-up.
 	ScaleUpStatusProcessor status.ScaleUpStatusProcessor
+	// NodeGroupManager is responsible for creating/deleting node groups.
+	NodeGroupManager nodegroups.NodeGroupManager
```

Comment:
A nitpick but adding a Manager to Processors suggests that maybe we should rename AutoscalingProcessors to AutoscalingExtensions or something more neutral than Processors.

Reply:
We can do that later.